assets/src/edit-story/components/form/dropDown/index.js

```diff
@@ -160,20 +186,12 @@ function DropDown({
 
 DropDown.propTypes = {
   value: PropTypes.any.isRequired,
-  options: PropTypes.array.isRequired,
   onChange: PropTypes.func.isRequired,
+  options: PropTypes.array,
   disabled: PropTypes.bool,
   lightMode: PropTypes.bool,
   placeholder: PropTypes.string,
   labelledBy: PropTypes.string,
 };
 
-DropDown.defaultProps = {
```

Comment:
What's the reason for not using defaultProps anymore?

Reply:
This component was using a mix of `defaultProps` and default values in component definition, which is a mess!

I decided to switch to only the latter, as it seems to be the recommended approach these days (and [will get deprecated at some point for functional components](https://twitter.com/dan_abramov/status/1133878326358171650?lang=en) anyway).